handler/handler.go

```diff
@@ -55,6 +60,7 @@ func (h *Handler) ListPosts(w http.ResponseWriter, r *http.Request) {
 	writeResponse(w, r, postList)
 }
 
+// Need comment documentation here
```

Comment:
ummm, you are currently on progress right?

Reply:
no, somebody else should done that lul

Reply:
https://github.com/helloproclub/cassiopeia/issues/2